users/forms.py

```diff
@@ -1,4 +1,5 @@
-from django.contrib.auth.forms import UserChangeForm
+from django.contrib.auth import authenticate
+from django.contrib.auth.forms import UserChangeForm, AuthenticationForm
```

Comment:
Please split imports into separate lines. 

This is specific to CodePoets code style, which, if you have not yet a chance to get familiar with, is located here https://github.com/Code-Poets/coding-style/wiki/python-style#zawijanie-wci%C4%99cia-nawiasy. If you don't have access, please let me know in comment. Please read it when you have time and try to follow it. Off course it is almost impossible for any developer regardless experience level to read it only once and follow it without any forgets, so nobody is expecting that from you. But if at some point you are not sure what to do, you should check there first for reference.